UPGRADE-2.x.md

```diff
@@ -1,2 +1,11 @@
 UPGRADE 2.x
 ===========
+
+UPGRADE FROM 2.0 to 2.1
+=======================
```

Comment:
Nope! You must put your upgrade note below `UPGRADE 2.x`. The `UPGRADE FROM X.X to X.X` is added by the maintainer that create the release.